MAVProxy/modules/mavproxy_chat/chat_openai.py

```diff
@@ -228,7 +234,7 @@ def handle_function_call(self, run):
                     # convert to json
                     stage_str = "convert output to json"
                     output = json.dumps(output)
-                except:
+                except Exception:
```

Comment:
This is a sensible change, but note that catching `Exception` can lead to very annoying behaviours where it catches things that you don't want it to (e.g. the user pressing ctrl-c to kill the program!).

Best to narrow exceptions as much as possible.